packages/router/src/history.tsx

```diff
@@ -7,19 +7,19 @@ const createHistory = () => {
     listen: (listener: Listener) => {
       const listenerId = 'RW_HISTORY_LISTENER_ID_' + Date.now()
       listeners[listenerId] = listener
-      window?.addEventListener('popstate', listener)
+      global.addEventListener('popstate', listener)
```

Comment:
Is it ok for these to not use the optional chaining operator `?.`

Reply:
When is `global` undefined?

Reply:
I meant for the execution of the functions; `global.addEventListener?.()`

Reply:
Yeah, should be fine. We're not triggering any navigations when prerendering, are we? So, even using `window` would have been fine. But to be consistent I'll use `global`